sound/soc/sof/intel/hda-codec.c

```diff
@@ -22,6 +22,7 @@
 
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC)
 #define IDISP_VID_INTEL	0x80860000
+#define CODEC_PROBE_RETRIES 3
```

Comment:
nit-pick: several typos in commit message

we often experience the
legacy HDA driver
legacy HDA driver, but it didn't help.

Reply:
Fixed all of them. Thanks.